src/marqo/s2_inference/languagebind/audio/modeling_audio.py

```diff
@@ -569,9 +569,9 @@ def forward(
         >>> from transformers import AutoTokenizer, CLIPTextModel
 
         >>> model = CLIPTextModel.from_pretrained("openai/clip-vit-base-patch32")
-        >>> tokenizer = AutoTokenizer.from_pretrained("openai/clip-vit-base-patch32")
+        >>> _tokenizer = AutoTokenizer.from_pretrained("openai/clip-vit-base-patch32")
```

Comment:
Is this due to a renaming using IDE's refactoring feature? Is this change accidental?